tox.ini

```diff
@@ -33,6 +33,9 @@ setenv =
     dev: MAILCHIMP_DIGESTS_SUBACCOUNT = {env:MAILCHIMP_DIGESTS_SUBACCOUNT:devdata}
     dev: MAILCHIMP_DIGESTS_EMAIL = {env:MAILCHIMP_DIGESTS_EMAIL:noreply-dev-2QY7xVpN@hypothes.is}
     dev: MAILCHIMP_DIGESTS_NAME = {env:MAILCHIMP_DIGESTS_NAME:Hypothesis (Dev)}
+    dev: MAILCHIMP_ANNOTATION_ACTIVITY_SUBACCOUNT = {env:MAILCHIMP_ANNOTATION_ACTIVITY_SUBACCOUNT:devdata}
```

Comment:
We don't need a new subaccount in dev, passing the new settings but with the same values.